vyper/semantics/types/module.py

```diff
@@ -323,16 +323,23 @@ def __init__(self, module: vy_ast.Module, name: Optional[str] = None):
 
         for i in self.import_stmts:
             import_info = i._metadata["import_info"]
-            self.add_member(import_info.alias, import_info.typ)
 
             if hasattr(import_info.typ, "module_t"):
-                self._helper.add_member(import_info.alias, TYPE_T(import_info.typ))
+                module_info = import_info.typ
+                # get_expr_info uses ModuleInfo
+                self.add_member(import_info.alias, module_info)
+                # type_from_annotation uses TYPE_T
+                self._helper.add_member(import_info.alias, TYPE_T(module_info.module_t))
+            else:  # interfaces
+                self.add_member(import_info.alias, TYPE_T(import_info.typ))
```

Comment:
can we add an assert for it being an interface?
might be useful in the future should we allow eg selective imports

Reply:
2d5f67df0e18e506918965a84722363b169c913e